adobe_vipm/flows/sync/asset.py

```diff
@@ -79,6 +64,17 @@ def _get_assets_for_update(self) -> list[dict]:
 
         return for_update
 
+    def _get_adobe_subscription(self, asset: dict[str, Any]) -> dict[str, Any] | None:
+        adobe_subscription_id = asset["externalIds"].get("vendor")
+        if not adobe_subscription_id:
+            logger.warning("No vendor found for asset %s", asset["id"])
```

Comment:
`No vendor subscription found...` - could you pls make it a little bit more explicit here. And I also add an explanation, that `asset.externalIds.vendor is empty`